main.go

```diff
@@ -626,5 +629,11 @@ func main() {
 		runDebugServer(ctx)
 	}()
 
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		cleaner.Run(ctx, 10*time.Minute, 30*time.Minute)
```

Comment:
I'd define these intervals as constants.

Reply:
+1

Reply:
I proposed to do the same:

> I also think X and Y should be _named_ constants until we really have a usecase for them being configurable.